friends/src/App.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import axios from 'axios'
+import Friend from './Friend/Friend';
+import NewFriend from './NewFriend/NewFriend';
+import './App.scss';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      friends: [],
+      postRes: '',
+      friend: {
+        name: '',
+        age: 0,
+        email: ''
+      }
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(res => this.setState({
+        friends: res.data
+      }))
+      .catch(err => console.log(err))
+  }
+
+  handleChanges = e => {
+    this.setState({
+      friend: {
+        ...this.state.friend,
+        [e.target.name]: e.target.value
+      }
+    })
+  }
+
+  postFriend = friend => {
+    axios
+      .post('http://localhost:5000/friends', friend)
+      .then(res => this.setState({
+        postRes: res.data.successMessage
+      }))
+      .catch(err => console.log(err))
+  }
+
+  newFriend = () => {
+    this.postFriend(this.state.friend)
+  }
```

Comment:
Why not combine `newFriend` with `postFriend` by changing line 41 to:
```
   .post('http://localhost:5000/friends', this.state.friend)
```
and omit the argument for postFriend all together?